Update enforcing-ssl.md (#22380)
dotnet/AspNetCore.Docs: aspnetcore/security/enforcing-ssl.md

The preceding approach is a one time operation per certificate and per WSL distribution. It's easier than exporting the certificate over and over. If you update or regenerate the certificate on windows, you might need to run the preceding commands again.

> ## Troubleshoot certificate problems such as certificate not trusted and clean fails

Author: ```suggestion
<a name="tcp"></a>

## Troubleshoot certificate problems such as certificate not trusted
```
